app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -26,7 +26,9 @@ const Set<String> kNeedsCQLabelList = <String>{'flutter/flutter'};
 const Set<String> kNeedsCheckGoldenTriage = <String>{'flutter/flutter'};
 
 /// List of repos that require check for labels and tests.
-const Set<String> kNeedsCheckLabelsAndTests = <String>{'flutter/flutter'};
+const Set<String> kNeedsCheckLabelsAndTests = <String>{'flutter/flutter', 'flutter/engine'};
+
+final RegExp kEngineTestRegExp = RegExp(r'tests?\.(dart|java|mm|m|cc)$');
```

Comment:
this is not the only place tests live - for example, C++ tests live along side their counterparts, conventionally named `*_unittests.cc`.

Reply:
this covers that case no?

Reply:
https://github.com/flutter/cocoon/pull/792/files#diff-304fad7784d8986a3af98d8752b20bc4R502 is what you meant right? (added a specific test for that just to make sure I'm not going crazy)

Reply:
Ahh yeah, ok. I misread the regexp.